internal/presenter/packages/spdx_json_presenter.go

```diff
@@ -48,9 +49,12 @@ func newSPDXJsonDocument(catalog *pkg.Catalog, srcMetadata source.Metadata) spdx
 		name = srcMetadata.Path
 	}
 
+	spdxDocumentID := spdx22.ElementID("DOCUMENT").String()
```

Comment:
We're not using this anymore in the return. It was part of old relationship code. I'll restore it.

Reply:
removed